src/azure-cli-core/azure/cli/core/parser.py

```diff
@@ -418,7 +392,6 @@ def _check_value(self, action, value):  # pylint: disable=too-many-statements, t
                     command_str = roughly_parse_command(cmd_list[1:])
                     ext_name = self._search_in_extension_commands(command_str)
                     if ext_name:
-                        caused_by_extension_not_installed = True
```

Comment:
Can you override the recommended command if caused_by_extension_not_installed is true? The output is like this if users type n for dynamic extension install.

>  $ az blueprint list
> The command requires the extension blueprint. Do you want to install it now? The command will continue to run after the extension is installed. (Y/n): n
> CommandNotFoundError: The command requires the latest version of extension blueprint. To install, run 'az extension add --upgrade -n blueprint'.
> Try this: 'az account list'
> Still stuck? Run 'az --help' to view all commands or go to 'https://docs.microsoft.com/en-us/cli/azure/reference-index?view=azure-cli-latest' to learn more